tailwind.config.js

```diff
@@ -2,7 +2,29 @@
 module.exports = {
   content: ['./src/**/*.{js,ts,jsx,tsx}'],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        neuzeit: 'Neuzeit S LT Std',
+      },
+      colors: {
+        'black-chinese': '#15151A',
+      },
+      fontSize: {
+        small: ['15px', '20px'],
+        regular: ['17px', '22px'],
+      },
+      backgroundImage: {
+        'sign-in': "url('assets/images/illustrations/background-sign-in.png')",
+      },
+      letterSpacing: {
+        'survey-tighest': '-1px',
```

Comment:
hopefully, we don't have to invent any more `tight`-adjectives lol 😆 
btw  the naming [goes with the tailwind world](https://tailwindcss.com/docs/letter-spacing) as well 👍🏿

Reply:
@tyrro I checked the Figma and added all customized letter spacings here, so that's all 😄